doc/citing.rst

```diff
@@ -8,22 +8,11 @@ please acknowledge this fact by citing `J. D. Hunter, "Matplotlib: A 2D
 Graphics Environment", Computing in Science & Engineering, vol. 9, no. 3,
 pp. 90-95, 2007 <https://doi.org/10.1109/MCSE.2007.55>`_.
 
-.. code-block:: bibtex
+.. literalinclude:: MCSE.2007.55.bib
```

Comment:
```suggestion
.. literalinclude:: MCSE.2007.55.bib
   :language: none
```
otherwise it gets python highlighting.

Reply:
Then you should set the language to bib:
https://pygments.org/docs/lexers/#lexers-for-bibtex-bibliography-data-and-styles

Reply:
Nice 👍  I set the language option as a new commit. Tested it, and the highlighting looks better. Both "bib", and "bibtex" keyword works. Went for the full name.